backend/src/main/java/de/sambam/gamenighttracker/service/UserService.java

```diff
@@ -19,11 +23,70 @@ public UserService(UserDb userDb) {
         this.userDb = userDb;
     }
 
-    public Optional<List<Game>> listAllGames(String id) {
+    public List<Game> listAllGames(String id) {
```

Comment:
I think the naming can be improved. Maybe something like ´listAllGameByPlayer´ and/or rename `id` to `userId`?